applications/luci-app-shadowsocks-libev/htdocs/luci-static/resources/shadowsocks-libev.js

```diff
@@ -67,6 +67,10 @@ function ucival_to_bool(val) {
 	return val === 'true' || val === '1' || val === 'yes' || val === 'on';
 }
 
+function startsWith(str, search) {
```

Comment:
This could be dropped if you replace it with `str.indexOf(search) === 0` checks.